lib/valueflow.cpp

```diff
@@ -4214,6 +4243,32 @@ static bool hasContainerSizeGuard(const Token *tok, unsigned int containerId)
     return false;
 }
 
+static bool isContainerSize(const Token* tok)
+{
+    if (!Token::Match(tok, "%var% . %name% ("))
```

Comment:
The day we want to handle structs/arrays etc we will have to change this so it uses the AST. I am not sure how we want it to work then so I think we should keep it this way.